WebContent/js/components/JobFile.js

```diff
@@ -36,18 +40,59 @@ class JobFile extends React.Component {
         }
     }
 
+    downloadJobFile() {
+        const { job, file, dispatch } = this.props;
+        atlasFetch(`jobs/${job.get('jobName')}/${job.get('jobId')}/files/${file.id}/content`, { credentials: 'include' })
```

Comment:
I chose to leave the fetch in the component rather than going to the effort of pushing content through Redux and the store as we shouldn't need to keep track of the content once it's downloaded

Reply:
agreed, tracking is not necessary